optaplanner-operator/optaplanner-operator-impl/src/k8s/school-timetabling-solver.yml

```diff
@@ -18,7 +18,8 @@ spec:
     spec:
       containers:
         - name: school-timetabling
-          image: quay.io/example/school-timetabling:latest
```

Comment:
How does this file relate to the IT? I thought that the IT works only with the demo copy.

Reply:
It is not , I just made minimal intrusion in school-timetable.Ok, let me make a lite version of it

Reply:
The image namespace is `example` so that users clearly understand there is no such image provided by OptaPlanner. If the namespace was `optaplanner`, they might look for a non-existing image there.